src/controls/TextControl.tsx

```diff
@@ -44,7 +45,9 @@ export function TextControl({
     (uischema.options as TextControlOptions) ?? {}
   const formItemProps =
     "formItemProps" in uischema ? uischema.formItemProps : {}
-  const tooltip = options.tooltip ? options.tooltip : formItemProps?.tooltip
+  const { tooltip, ...formItemPropsWOTooltip } = formItemProps as FormItemProps
+  const mergedTooltip = options.tooltip ? options.tooltip : tooltip ?? ""
```

Comment:
it's not really merged, maybe just call this `tooltip` and alias the tooltip from formItemProps as `formItemTooltip`?